### PR TITLE
enhance: Revert default l0 forward policy to FilterByBF

### PR DESCRIPTION
**Pull request by congqixia**
Related to #37767

### CODE REVIEW

**Comment by congqixia** [comment]
rerun cpp-unit-test

File: configs/milvus.yaml [hunk 1]
```diff
@@ -748,7 +748,7 @@ msgChannel:
 log:
   # Milvus log level. Option: debug, info, warn, error, panic, and fatal. 
   # It is recommended to use debug level under test and development environments, and info level in production environment.
-  level: info
+  level: debug
```
Comment (czs007) [inline configs/milvus.yaml:751]: revert this

**Comment by congqixia** [comment]
rerun cpp-unit-test

**Pull request merged**